Wrappers/Python/cil/plugins/tigre/FBP.py

```diff
@@ -82,6 +82,9 @@ def check_input(self, dataset):
 
         DataOrder.check_order_for_engine('tigre', dataset.geometry)
         return True
+    
+    def check_output(self, out):
+        return True
```

Comment:
I'm not sure why you're skipping the check here?

Reply:
I'm not sure! I've updated this now.